p4-16/spec/grammar.mdk

```diff
@@ -178,14 +178,22 @@ keysetExpression
     ;
 
 tupleKeysetExpression
-    : '(' simpleKeysetExpression ',' simpleExpressionList ')'
+    : "(" simpleKeysetExpression "," simpleExpressionList ")"
+    | "(" reducedSimpleKeysetExpression ")"
     ;
 
 simpleExpressionList
     : simpleKeysetExpression
     | simpleExpressionList ',' simpleKeysetExpression
     ;
 
+reducedSimpleKeysetExpression
+    : expression "&&&" expression
+    | expression ".." expression
+    | DEFAULT
+    | "_"
+    ;
+
```

Comment:
If there is a reason to introduce reducedSimpleKeysetExpression, it seems reasonable for humans reviewing and understanding this to make its definition as similar to simpleKeysetExpression as possible, e.g. something like "DEFAULT | DONTCARE | expression MASK expression | expression RANGE expression"